layout.yaml

```diff
@@ -12,25 +12,25 @@ teams:
     display_name: Level 3 South
     # T_6 to T_13 on the floorplans
```

Comment:
Are these comments still meaningful?

Reply:
We've not changed the floorplan. Probably easier to delete